lib/diagnostic.rb

```diff
@@ -7,36 +7,36 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.split(' ').max_by(&:length)
   end
```

Comment:
Very close. What does `.split` work with? is list the appropriate datatype?

What does `list.split(' ')` give you? is that the appropriate datatype on which to use `max_by`? take a look at the documentation for `.max_by`: http://ruby-doc.org/core-2.3.1/Enumerable.html#method-i-max_by